src/actions/index.js

```diff
@@ -30,7 +31,9 @@ export function getResponseJSON(response) {
 export function requestErrorHandler(dispatch, fetchAction) {
   return (err) => {
     const callName = fetchAction ? fetchAction.type : "";
-    notifyError("API-kutsu " + callName + " epäonnistui: " + err + ". Kokeile pian uudelleen.");
+    notifyError(getMessage("APICall") + " " + callName + " " +
```

Comment:
I think this message oughta be one string:
```javascript
{
  apiCallFailed: 'API-kutsu {callName} epäonnistui: {err}. ....',
}
```
and the placeholders replaced here:
```javascript
notifyError(getMessage('apiCallFailed').replace('{callName}', callName).replace('{err}', err));
```